### PR TITLE
Add Windows installation

### PR DESCRIPTION
**Pull request by mabelzhang**
Partially addresses https://github.com/ignitionrobotics/docs/issues/117

Made headings and code blocks consistent with other per-package installation tutorials.

DART will need to be resolved, see https://github.com/ignitionrobotics/docs/issues/96

### CODE REVIEW

**Comment by traversaro** [comment]
As I guess many users of ignition-physics would probably be expecting to be able to use DART on Windows, it could make sense to mention that DART does not support Windows at the moment (see https://github.com/dartsim/dart/issues/1522 and https://github.com/conda-forge/dartsim-feedstock/issues/2 .

**Comment by mabelzhang** [comment]
Thanks! Just realized and came back to edit the post.

File: tutorials/02_installation.md [hunk 1]
```diff
@@ -1,127 +1,194 @@
...
 cmake ..
 make
 sudo make install
 ```
+
+## Windows
+
+### Prerequisites
+
+First, follow the [ign-cmake](https://github.com/ignitionrobotics/ign-cmake) tutorial for installing Conda, Visual Studio, CMake, etc., prerequisites, and creating a Conda environment.
```
Comment (JShep1) [inline tutorials/02_installation.md:137]:
```suggestion
First, follow the [ign-cmake](https://github.com/ignitionrobotics/ign-cmake) tutorial for installing Conda, Visual Studio, CMake, and other prerequisites, and also for creating a Conda environment.
```

**Review by JShep1 — COMMENTED** [review]
I agree we should explicitly state dart currently isn't supported, perhaps with links to the outstanding issues in case anyone wants to lend a hand :smile: .  We should also probably state that only TPE is supported currently for Windows.

**Comment by mabelzhang** [comment]
Closing in favor of #194, where comments above have been addressed

**Pull request closed without merging**